components/org.wso2.carbon.um.ws.service/src/main/resources/META-INF/services.xml

```diff
@@ -35,6 +35,210 @@
+            </parameter>
+        </operation>
+
+        <operation name="getRoleListOfUser">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/rolemgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getRoleNames">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getTenantId">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getTenantIdofUser">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserClaimValue">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserClaimValues">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserClaimValuesForClaims">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserId">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="getUserList">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/list
+            </parameter>
+        </operation>
+
+        <operation name="getUserListOfRole">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/rolemgt/view
+            </parameter>
+        </operation>
+
+        <operation name="isExistingRole">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/rolemgt/view
+            </parameter>
+        </operation>
+
+        <operation name="isReadOnly">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/view
+            </parameter>
+        </operation>
+
+        <operation name="listUsers">
+            <parameter name="AuthorizationAction"
+                       locked="true">/permission/admin/manage/identity/usermgt/list
```

Comment:
should this permission be /permission/admin/manage/identity/usermgt/view as well?

Reply:
Agreed in mail thread